.vscode/settings.json

```diff
@@ -6,7 +6,9 @@
     "typescriptreact"
   ],
   "editor.formatOnSave": true,
-  "eslint.enable": true,
```

Comment:
This setting is deprecated and the current configuration was non sufficient to enable format on save. The `codeActionsOnSave` option follows the same config we expose on the CLI.
<img src="https://user-images.githubusercontent.com/17585917/103819682-e2e54900-5038-11eb-9e33-93e3c6622c65.png" height=100>